modules/disconnected-osus-overview.adoc

```diff
@@ -7,16 +7,16 @@
 
```

Comment:
The changes in this file are LGTM, in addition to `local` OSUS, I see there are many `local registry`, for example:  `You must provision a local container image registry with the container images for your update, as described in Mirroring OpenShift Container Platform images.`. Per my understanding, we really don't need a `local registry`, I only need a registry can be accessed by OSUS, right? so maybe we need also update the description of `local registry`. cc @jiajliu @shellyyang1989

Reply:
Thanks for calling this out, I was curious about this myself as well. Can anyone confirm that this is true? If so, I can make similar wording changes to the parts of this doc that mention registries.

CC @petr-muller and @wking

Reply:
Yeah, that is correct. A model disconnected environment would have a image registry that would serve as a canonical source of images for that environment. For some reason "local" for *me* does not imply "present on the cluster", likely because we usually call the built-in registry an "internal registry" but unifying how we talk about these systems sounds good to me. Something like "You must provision a container image registry in your environment with the container images..."

Reply:
Good point @JianLi-RH